openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion/__init__.py

```diff
@@ -171,10 +172,10 @@ def student_view(self, context=None):
         """
         fragment = Fragment()
         if ENABLE_DISCUSSIONS_MFE.is_enabled(self.course_key) and settings.DISCUSSIONS_MICROFRONTEND_URL:
-            url = f"{settings.DISCUSSIONS_MICROFRONTEND_URL}/discussions/{self.course_key}/topics/{self.discussion_id}"
+            mfe_url = get_discussions_mfe_topic_url(self.course_key, self.discussion_id, in_context_view=True)
```

Comment:
I think this is wrong. `get_discussions_mfe_topics_url` accepts `view` parameter, not `in_context_view`. So you either need to change the signature of the function (and `get_discussions_mfe_url` function as well), or to pass `view="in_context"`.
I tried to find a way to reproduce the issue, but couldn't find a way in the reasonable amount of time.